CRM/Report/Form.php

```diff
@@ -4300,7 +4300,8 @@ public function selectedTables() {
         }
         if (array_key_exists('filters', $table)) {
           foreach ($table['filters'] as $filterName => $filter) {
-            if (!empty($this->_params["{$filterName}_value"])
+            if ((isset($this->_params["{$filterName}_value"])
+                && !CRM_Utils_System::isNull($this->_params["{$filterName}_value"]))
```

Comment:
@pradpnayak or : 
```php
if (CRM_Utils_Rule::boolean($this->_params["{$filterName}_value"])) {
 ...
}
```
?

Reply:
Hi @monishdeb 
Correct me if am wrong, CRM_Utils_Rule::boolean() checks for 0,1,Yes,No values but in my case the value can be any. I guess isNull would be better option. 

Thoughts?

Pradeep

Reply:
Hmm right. Well, I have no issue with this change. 

Happy to merge 👍